docs/ingestion/native-batch.md

```diff
@@ -414,6 +381,29 @@ them to create the final segments. Finally, they push the final segments to the
 > in `partial dimension distribution` and `partial segment generation` phases,
 > the task may fail if the input changes in between the two passes.
 
+#### Multi-dimension range partitioning
+> Multiple dimension (multi-dimension) range partitioning is an experimental feature. Multi-dimension range partitioning is currently not supported in the sequential mode of the Parallel task.
+
+When you use multi-dimension partitioning for your data, Druid is able to distribute segment sizes more evenly than with single dimension partitioning.
+
+For segment pruning to be effective and translate into better query performance, you must the first of your `partitionDimensions` at query time. For example, given the following `partitionDimensions`:
```

Comment:
I think a word(s) is missing here, but not sure what it should be:  "...you must __ the first..."